webboot/webboot.go

```diff
@@ -117,12 +118,24 @@ func usage() {
 	os.Exit(1)
 }
 
+func setupWifi(wifi string) error {
```

Comment:
The function name is great, but change the variable wifi to essid

Reply:
Done